quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java

```diff
@@ -51,6 +51,5 @@ void failIfDevProfile(LaunchResult result) {
                 () -> "The Error Output:\n" + result.getErrorOutput() + "doesn't contains the expected string.");
         assertTrue(result.getErrorOutput().contains("For more details run the same command passing the '--verbose' option. Also you can use '--help' to see the details about the usage of the particular command."),
                 () -> "The Error Output:\n" + result.getErrorOutput() + "doesn't contains the expected string.");
-        assertEquals(4, result.getErrorStream().size());
```

Comment:
This is actually making sure we don't introduce regressions. Change tests like that usually is not a good idea for this reason.

Or do you have something better in mind?

Our requirement is to make sure we have specific messages are shown only once.

Reply:
I'll try to have a more generic test for this 👍

Reply:
@pedroigor should be ok now 🙂